irs-api/src/main/java/org/eclipse/tractusx/irs/connector/job/MultiTransferJob.java

```diff
@@ -106,6 +106,10 @@ public JobParameter getJobParameter() {
         return getJob().getParameter();
     }
 
+    public boolean jobIsCompleted() {
+        return this.getJob().getState().equals(JobState.COMPLETED);
```

Comment:
Please think about this if this is good style.

Reply:
Yes, it is. It is a method which does exactly what is says which makes the code where this is used more readable.